block_darwin.go

```diff
@@ -0,0 +1,250 @@
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package ghw
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
You will need to add this dependency to Gopkg.toml as well. Alternately, consider just using `fmt.Errorf()` everywhere instead of the github.com/pkg/errors library.